### ISSUE TITLE
The interface of bitocracy.sovryn.app is showing twice a vote when it is really different votes

### ISSUE THREAD

**Issue by jjmr007** [DistributedCollective/governance-dapp#33]
**Describe the bug**

1.- A user observed that some votes are being showed twice in the UI. It is appearing in a random fashion.

2.- The UI is also showing the wrong totalization of votes

3.- The column of "Addresses" and "Transaction Ids" are showing the same data and the same URL: the address of the voter.

**To Reproduce**
Take a look on the voting lists on the bitocracy.sovryn.app web.

**Expected behavior**
A vote should be shown only once by each SIP, by the UI.

**Screenshots**
This is an image of SIP-0006a:
![UI_triggers_2_votes](https://user-images.githubusercontent.com/17028457/109330417-030fd800-7832-11eb-8f94-7dd672bf73bc.png) 

And this an image for SIP-0008
![same_in_the_other_SIP](https://user-images.githubusercontent.com/17028457/109330532-2175d380-7832-11eb-9e27-9ca618031ac1.png)

**Desktop (please complete the following information):**

The incident was observed from a Windows 10 PC, in Chrome browser

It may cause some concern among the users (it worried me, until I found it is a minor bug) so it is convenient to fix. Thanks

**Comment by creed-victor** [comment]
Duplicate of #28
